### DISCUSSION TITLE
In a cluster configuration, does the edge and origin group show the log files for all instances?

### DISCUSSION THREAD

**Discussion by tremzo** [ant-media/Ant-Media-Server#5653] [FAQs]
When I log in to the edge or origin group of my cluster, does the dashboard show the logs for all instances or just the instance I've logged into? 

And if I want to send those logs to a log analysis tool, would I need to install the software, for example, Promtail on all instances or just one?

**Comment by Mohit-3196** [comment]
Hi @timantmedia,
Thank you for your question.

When you log in to the dashboard of the cluster, the logs that it shows are related to the instance to which you connect.
If you want to access the logs of any particular node of the cluster, you can ssh into that node and collect the logs.

We also have ways to collect the cluster logs using Graylog and you can find more about it from here https://antmedia.io/docs/guides/advanced-usage/monitoring/collecting-logs-from-ams-cluster/

Thank you.
